### DISCUSSION TITLE
Protect from DPI feature in AdGuard Home

### DISCUSSION THREAD

**Discussion by PaperCutBad** [AdguardTeam/AdGuardHome#5687] [Ideas]
Is it possible and is there any plan to add DPI bypass feature like in AdGuard (Windows,Android)? For now I have to use spoof-dpi for now but then I am not able to use Adguard home on my Linux setup.

**Comment by ghost** [comment]
Yes please.
ISPs now use DPI for censoring so DoT, DoH and DNScrypt are useless.
Although I use AdguardHome on OpenWrt it seems useless due to DPI.
In the end I used PowerTunnel on my Linux machine.

**Comment by trmdi** [comment]
This feature is outside the scope of AGH. However, you could set up a transparent proxy like zapret on your router.
